src/Home/index.tsx

```diff
@@ -82,198 +86,263 @@ const Home: FunctionComponent = () => {
-        <View style={style.topScrollViewBackground} />
-        <GradientBackground gradient={Colors.gradient100} angleCenterY={1}>
-          <View style={style.topContainer}>
-            <SettingsButton />
-            <View style={style.topIconContainer}>
+        <Text style={style.headerText}>{t("screen_titles.home")}</Text>
+        <TouchableOpacity
+          style={style.settingsButtonContainer}
+          accessible
+          accessibilityLabel={t("home.open_settings")}
+          onPress={handleOnPressSettings}
+        >
+          <SvgXml
+            xml={Icons.Gear}
+            width={Iconography.small}
+            height={Iconography.small}
+            fill={Colors.neutral100}
+          />
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={statusContainerStyle}
+          accessibilityLabel={statusText}
+          testID={"exposure-scanning-status-button"}
+          onPress={handleOnPressExposureDetectionStatus}
+        >
+          <View style={style.statusTopContainer}>
+            <Text style={style.statusText} testID={"home-header"}>
+              {statusText}
+            </Text>
+            <View>
               <SvgXml
-                xml={topIcon}
-                width={TOP_ICON_SIZE}
-                height={TOP_ICON_SIZE}
-                fill={topIconFill}
-                accessible
-                style={style.topIcon}
-                accessibilityLabel={topIconAccessibilityLabel}
+                xml={statusIcon}
+                width={STATUS_ICON_SIZE}
+                height={STATUS_ICON_SIZE}
+                fill={statusIconFill}
+                style={style.statusIcon}
               />
-              {appIsActive && <ExpandingCircleAnimation />}
+              {exposureDetectionStatus && <ExpandingCircleAnimation />}
             </View>
-            <Text style={style.headerText} testID={"home-header"}>
-              {headerText}
-            </Text>
-            <Text style={style.subheaderText} testID={"home-subheader"}>
-              {subheaderText()}
-            </Text>
           </View>
-        </GradientBackground>
-        <View style={style.bottomContainer}>
-          <View>
-            <ShareLink />
-            <BluetoothActivationStatus />
-            <ProximityTracingActivationStatus />
-            <LocationActivationStatus />
-            {displayCovidData && <COVIDDataDashboard />}
+          <View style={style.statusBottomContainer}>
+            <Text style={style.statusActionText}>{actionText}</Text>
+            <SvgXml
+              xml={Icons.ChevronRight}
+              fill={Colors.black}
+              width={Iconography.tiny}
+              height={Iconography.tiny}
+            />
           </View>
-          <View style={style.ctaContainer}>
+        </TouchableOpacity>
+        <ShareLink />
+        {displayCovidData && <COVIDDataDashboard />}
+        {displayCallbackForm && (
+          <View style={style.floatingContainer}>
```

Comment:
What do you think about `<CallbackFormClip />`?

Reply:
👍